utils/spdx/src/main/kotlin/SpdxUtils.kt

```diff
@@ -179,3 +180,14 @@ private fun getLicenseTextFile(id: String, dir: File): File? =
     ).mapNotNull { filename ->
         dir.resolve(filename).takeIf { it.isFile }
     }.firstOrNull()
+
+/**
+ * Try to locate the directory which contains the license texts provided by ScanCode. This requires ScanCode to be
+ * installed and the `scancode` binary to be on the path.
+ */
+fun getScanCodeLicenseTextsDir(): File? {
+    val scanCodeDir = getPathFromEnvironment("scancode")?.parentFile?.absoluteFile
```

Comment:
I believe we can drop `?.absoluteFile` here, these paths should already be absolute.